src/commands/autocomplete/index.test.ts

```diff
@@ -10,76 +10,49 @@ cli.config.mock = true
 
 runtest('outputs install instructions for zsh', async () => {
   await Autocomplete.mock(['zsh'])
-  expect(cli.stdout.output).toMatch(`Setup Instructions for CLI-ENGINE CLI Autocomplete ---
-
-1) Add the autocomplete env var to your zsh profile
-
-$ printf "$(cli-engine autocomplete:script zsh)" >> ~/.zshrc
-
-2) Source your updated zsh profile
+  expect(cli.stdout.output).toMatch(`
```

Comment:
@crcastle new zsh/bash output (cache build has been moved to the top but you won't see it here in the test)